app/src/main/java/io/github/sustainow/MainActivity.kt

```diff
@@ -50,12 +80,111 @@ class MainActivity : ComponentActivity() {
+                        val canNavigateBack = previousBackStackEntry != null
+                                && previousScreen != Login
+                                && previousScreen != SignUp
+
+                        if (currentScreen != Login && currentScreen != SignUp) {
+                            TopAppBar(
+                                title = {
+                                    Text(text = context.getString(R.string.login_email_button_text), color = MaterialTheme.colorScheme.onPrimaryContainer)
+                                        val logoResource = painterResource(id = R.drawable.sustainow_logo_transparent)
+                                        Row(
+                                            modifier = Modifier.fillMaxWidth(),
+                                            verticalAlignment = Alignment.CenterVertically
+                                        ) {
+                                            Spacer(modifier = Modifier.weight(1f)) // Adiciona espaço entre o botão de voltar e a logo
+                                            // Tornar a logo clicável
+                                            Image(
+                                                logoResource,
+                                                contentDescription = null,
+                                                modifier = Modifier.requiredSize(150.dp, 150.dp)
+                                            )
+                                            Spacer(modifier = Modifier.weight(1f)) // Centraliza a logo
+                                        }
+                                },
+                                colors = TopAppBarDefaults.topAppBarColors(
+                                    containerColor = MaterialTheme.colorScheme.tertiaryContainer,
+                                ),
+                                navigationIcon = {
+                                    if(canNavigateBack) {
+                                        IconButton(onClick = {
+                                            navController.popBackStack()
+                                        }) {
+                                            Icon(
+                                                imageVector = Icons.AutoMirrored.Filled.ArrowBack,
+                                                contentDescription = context.getString(R.string.back)
+                                            )
+                                        }
+                                    }
+                                },
+                                actions = {
+                                    IconButton(onClick = { showUserMenu = !showUserMenu }) {
+                                        Icon(
+                                            Icons.Default.AccountCircle,
+                                            contentDescription = context.getString(R.string.user_menu)
+                                        )
```

Comment:
Ícone padrão é somente quando o usuário está logado, sequência de verificações é:
- `userState is Logged`, caso contrário não é preciso renderizar esse botão
- `userState.user.profilePicture` é uma string não-vazia, se sim renderizar uma imagem com o mesmo tamanho do ícone, se não renderizar o ícone padrão